Source/TestDafny/MultiBackendTest.cs

```diff
@@ -74,10 +76,20 @@ public class MultiBackendTest {
     // but this was never meaningful and only added maintenance burden.
     // Here we only ensure that the exit code is 0.
 
+    // We also use --(r|b)print to catch bugs with valid but unprintable programs.
```

Comment:
That's exactly the intention, it's another feature that should work for every valid Dafny program but where historically there have been bugs. It's particularly valuable since the current `.doo` file format relies on printing programs as well now. Doing this for every test is the simplest way to make sure existing tests that use these options for the same testing motivation still provide at least as much testing coverage.

I don't think there's added complexity for test authors, but there is a runtime cost when running the tests. As per your other comment I'll evaluate that if it seems expensive.